src/MOI_wrapper.jl

```diff
@@ -433,3 +434,14 @@ end
     return sparse(Int.(qmatind), qmatcol, qmatval)
  end
 
+function LQOI.make_problem_type_integer(optimizer::Optimizer)
+    optimizer.inner.has_int = true
+    set_prob_type!(optimizer.inner, :MILP)
```

Comment:
You need to check for quadratics as well. Check the MPB code. I think there are helper functions.

Reply:
Ok. Tests are all screwed-up for now

Reply:
and by chance they fail only for non-linear stuff